### PR TITLE
Fix grid model hosting for large models

### PR DESCRIPTION
**Pull request by IonesioJunior**
## Description
Due to the need to support uploads for large models **_(>=200MB)_**, the approach used by the `serve_model` method has been modified. We changed the use of standard WebSockets messages to a well-known solution used for uploading large files on HTTP servers.

Fix [#23](https://github.com/OpenMined/PyGridNode/issues/23) 

## Affected Dependencies
- **requests_toolbelt**

## How has this been tested?
```
import syft as sy
import torch as th
from syft.workers.node_client import NodeClient
hook = sy.TorchHook(th)

node = NodeClient("ws://<node_address>")

traced_model = th.jit.load('large_traced_model.jit')

node.serve_model(traced_model,
                 model_id="myCustomModel",
                 allow_download=True,
                 allow_remote_inference=True)
```

## Checklist
- [X] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have commented on my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [X] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: syft/workers/node_client.py [hunk 1]
```diff
@@ -248,10 +250,31 @@ def serve_model(
...
+            "model": serialized_model,
         }
-        response = self._forward_json_to_websocket_server_worker(message)
-        return self._return_bool_result(response)
+
+        url = self.address.replace("ws", "http") + "/serve-model/"
+
+        # Multipart encoding
+        form = MultipartEncoder(message)
+        upload_size = form.len
```
Comment (shubham3121) [inline syft/workers/node_client.py:260]: Can `form.len` be `None` ? If so, can be put a check and maybe raise a error of message size being null.
